server/pom.xml

```diff
@@ -482,6 +488,7 @@
         <ssl.address.version>${project.version}</ssl.address.version>
```

Comment:
Not related this check-in, but I think we can get rid of all these properties as they are all reside in the same project and released together.